### DISCUSSION TITLE
System Requirements

### DISCUSSION THREAD

**Discussion by DoS007** [danny-avila/LibreChat#3941] [Q&A]
Hi there,

How much ram does a running docker installation need?

**Comment by kukoboris** [comment]
Minimum Requirements

The minimum requirements for running LibreChat:

    1 GiB RAM
    1 vCPU

Note: With everything enabled, you might consider increasing the RAM to 2GB for smoother operation.

https://www.librechat.ai/docs/remote

**Reply by DoS007** [reply]
Thank you. It's probably meant as for a whole system only running librechat.
